.eslintrc

```diff
@@ -1,4 +1,5 @@
 {
+	"root": true,
```

Comment:
This prevents `eslint` from searching above this directory for configuration. It doesn't actually limit it to the current directory, but anything below the current directory.